Tests/TuistLoaderTests/Up/UpMintTests.swift

```diff
@@ -0,0 +1,158 @@
+            }
+            XCTAssertEqual(error.description, "Mintfile not found at path \(temporaryPath.pathString)")
+            XCTAssertEqual(error.type, .abort)
+        }
+    }
+
+    func test_isMet_when_mintfile_is_empty() throws {
+        let subject = UpMint(linkPackagesGlobally: false, upHomebrew: upHomebrew)
+        let temporaryPath = try self.temporaryPath()
+        let mintfile = temporaryPath.appending(component: "Mintfile")
+        try fileHandler.touch(mintfile)
+
+        upHomebrew.isMetStub = { _ in true }
+        system.succeedCommand(["cat", "\(mintfile.pathString)"])
+
+        XCTAssertTrue(try subject.isMet(projectPath: temporaryPath))
+    }
+
+    func test_isMet_when_mint_packages_are_installed() throws {
+        let subject = UpMint(linkPackagesGlobally: false, upHomebrew: upHomebrew)
+        let temporaryPath = try self.temporaryPath()
+        let mintfile = temporaryPath.appending(component: "Mintfile")
+        try fileHandler.touch(mintfile)
+        let package = "apple/swift-format@swift-5.1-branch"
+
+        upHomebrew.isMetStub = { _ in true }
+        system.succeedCommand(["cat", "\(mintfile.pathString)"], output: package)
+        system.succeedCommand(["mint", "which", "\(package)"])
+
+        XCTAssertTrue(try subject.isMet(projectPath: temporaryPath))
+    }
+
+    func test_isMet_when_mint_packages_are_not_installed() throws {
+        let subject = UpMint(linkPackagesGlobally: false, upHomebrew: upHomebrew)
+        let temporaryPath = try self.temporaryPath()
+        let mintfile = temporaryPath.appending(component: "Mintfile")
+        try fileHandler.touch(mintfile)
+        let package = "apple/swift-format@swift-5.1-branch"
+
+        upHomebrew.isMetStub = { _ in true }
+        system.succeedCommand(["cat", "\(mintfile.pathString)"], output: package)
+        system.errorCommand(["mint", "which", "\(package)"])
+
+        XCTAssertFalse(try subject.isMet(projectPath: temporaryPath))
+    }
+
+    func test_meet_when_homebrew_is_not_met() throws {
+        let subject = UpMint(linkPackagesGlobally: false, upHomebrew: upHomebrew)
+        let temporaryPath = try self.temporaryPath()
+        let mintfile = temporaryPath.appending(component: "Mintfile")
+        try fileHandler.touch(mintfile)
+        upHomebrew.isMetStub = { _ in false }
+
+        upHomebrew.meetStub = { projectPath in
+            XCTAssertEqual(temporaryPath, projectPath)
+        }
+        system.succeedCommand(["mint", "bootstrap", "-m", "\(mintfile.pathString)"])
+
+        try subject.meet(projectPath: temporaryPath)
+
+        XCTAssertEqual(upHomebrew.meetCallCount, 1)
+    }
+
+    func test_meet_when_mintfile_doesnt_exist() throws {
+        let subject = UpMint(linkPackagesGlobally: false, upHomebrew: upHomebrew)
+        let temporaryPath = try self.temporaryPath()
+
+        upHomebrew.isMetStub = { _ in true }
+
+        XCTAssertThrowsError(try subject.isMet(projectPath: temporaryPath)) { error in
```

Comment:
Same comment here in regards to simplifying the assertion and not testing the error description and type as part of this test.